src/OrbitGl/TimerTrack.cpp

```diff
@@ -81,22 +81,19 @@ void TimerTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick,
   // events that would just draw over an already drawn line. When zoomed in
   // enough that all events are drawn as boxes, this has no effect. When zoomed
   // out, many events will be discarded quickly.
-  uint64_t min_ignore = std::numeric_limits<uint64_t>::max();
-  uint64_t max_ignore = std::numeric_limits<uint64_t>::min();
   uint64_t time_window_ns = static_cast<uint64_t>(1000 * time_graph_->GetTimeWindowUs());
   uint64_t pixel_delta_in_ticks = time_window_ns / canvas->GetWidth();
   uint64_t min_timegraph_tick = time_graph_->GetTickFromUs(time_graph_->GetMinTimeUs());
 
   for (auto& chain : chains_by_depth) {
     if (!chain) continue;
-    for (TimerChainIterator it = chain->begin(); it != chain->end(); ++it) {
-      TimerBlock& block = *it;
+    for (auto& block : *chain) {
       if (!block.Intersects(min_tick, max_tick)) continue;
 
       // We have to reset this when we go to the next depth, as otherwise we
       // would miss drawing events that should be drawn.
```

Comment:
This comment says that `min_ignore` and `max_ignore` start fresh when moving to a new depth, but shouldn't they be in the outer loop for that? Here they are reset for each block of the chain of each depth. I'm confused (but feel free to ignore, this is unrelated to this change).